src/trace-plugin-loader.js

```diff
@@ -95,6 +92,11 @@ function activate(agent) {
         }
         instrumentation.patches[moduleRoot] = patchSet;
       }
+
+      // Create a new object exposing functions to create trace spans and
+      // propagate context. This relies on functions currently exposed by the
+      // agent.
+      var api = new PluginAPI(agent, moduleRoot);
```

Comment:
Once this change is made, `test-grpc-context.js` will fail because it's checking the second argument of `logger.debug`, which now doesn't contain a value.